### ISSUE TITLE
Hat Visible/Weapon Visible Issue

### ISSUE THREAD

**Issue by SerenitySunfire** [Ottermandias/Glamourer#25]
All of my designs have Hat Visible and Weapon Visible unchecked, but since 6.1, it is not respecting my choices and flagging them on even though the design says not to.

**Comment by Rueby** [comment]
Same problem

**Comment by Ottermandias** [comment]
I think the checkboxes for Hat Visible and Weapon Visible are inverted at the moment. Visor Toggled and Is Wet should be correct.
Can you test using them invertedly?

**Comment by SerenitySunfire** [comment]
Yep. Looks like they are. I checked the box for them, and now they disappeared.
